contents/docs/experiments/experiment-significance.mdx

```diff
@@ -0,0 +1,103 @@
+
+#### 1. Probability of being the best
+
+We use Monte Carlo simulations to determine the probability of each variant being the best. Each variant can be modeled as a beta distribution, with the alpha parameter equal to the number of conversions and the beta parameter equal to the number of failures for that variant. For each variant, we sample from their respective distributions to get a conversion rate. We perform 100,000 simulation runs in our calculations.
+
+The probability of a variant being the best is given by:
+
+<FormulaScreenshot
+  imageLight={FunnelExperimentCalculationLight}
+  imageDark={FunnelExperimentCalculationDark}
+  alt="Funnel experiment calculation" 
+  classes="rounded"
+/>
+
+#### 2. Statistical significance
+
+To calculate significance, we measure the expected loss, as described in [VWO's SmartStats whitepaper](https://vwo.com/downloads/VWO_SmartStats_technical_whitepaper.pdf).
+
+To do this, we run a Monte Carlo simulation and calculate the loss as:
+
+<FormulaScreenshot
+  imageLight={FunnelSignificanceLight}
+  imageDark={FunnelSignificanceDark}
+  alt="Funnel significance" 
+  classes="rounded"
+/>
+
+This represents the expected loss in conversion rate if you choose any other variant. If this loss is below 1%, we declare the results significant.
+
+## Trend experiment calculations
+
+Trend experiments capture count data. For example, if you want to measure the change in the total count of clicks, you would use this type of experiment.
+
+#### 1. Probability of being the best
+
+We use Monte Carlo simulations to determine the probability of each variant being the best. Each variant can be modeled as a gamma distribution, with the shape parameter equal to the trend count and the exposure parameter equal to the relative exposure for that variant. For each variant, we sample from their respective distributions to get a count value. We perform 100,000 simulation runs in our calculations.
+
+The probability of a variant being the best is given by:
+
+<FormulaScreenshot
+  imageLight={TrendExperimentCalculationLight}
+  imageDark={TrendExperimentCalculationDark}
+  alt="Trend experiment calculation" 
+  classes="rounded"
+/>
+
+>**Trend experiment exposure**
+>
+>
+>Trend experiments compare counts of events. Since count data can refer to the total count of events or the number of unique users, we use a proxy metric to measure exposure. The number of times the `feature_flag_called` event returns control or test is used as the respective exposure for the variant. This event is sent automatically when you call `posthog.getFeatureFlag()`.
+>
+>It's important to note that a variant showing fewer count data can still have a higher probability of being the best if its exposure is much smaller. This is because the relative exposure is taken into account when calculating probabilities.
```

Comment:
```suggestion
>Note that a variant showing fewer count data can still have a higher probability of being the best if its exposure is much smaller. This is because the relative exposure is taken into account when calculating probabilities.
```